sdk/ml/azure-ai-ml/tests/pipeline_job/e2etests/test_pipeline_job.py

```diff
@@ -1629,3 +1581,70 @@ def test_reused_pipeline_child_job_download(
         assert next(artifact_dir.iterdir(), None), "No artifacts were downloaded"
         assert output_dir.exists()
         assert next(output_dir.iterdir(), None), "No outputs were downloaded"
+
+
+@pytest.mark.usefixtures(
```

Comment:
maybe split this into a new test file

Reply:
split these test cases into a new test file